tests/foundry/test/Loops.t.sol

```diff
@@ -48,6 +48,27 @@ contract LoopsTest is Test {
         uint256 actual = sumToNBroken(n);
         assertEq(expected, actual);
     }
+
+    function sum_N(uint n) pure internal returns (uint) {
+        uint s = 0;
+        while (0 < n) {
+            s = s + n;
+            n = n - 1;
+        }
+        return s;
+    }
+
+    function test_sum_10() public returns (uint) {
+        return sum_N(10);
+    }
+
+    function test_sum_100() public returns (uint) {
+        return sum_N(100);
+    }
+
+    function test_sum_1000() public returns (uint) {
+        return sum_N(1000);
```

Comment:
Is the purpose of these tests to just check that KEVM is able to execute the loop to the end? We could also add an `assert` that the result is correct.